examples/tabs/js/alt.js

```diff
@@ -0,0 +1,4 @@
+var Alt = require('../../../src/alt')
+var alt = new Alt()
+
+module.exports = alt
```

Comment:
My opinion on this: I'm okay with keeping semicolons in examples and documentation so as to keep with the larger norm in the JS community, but any alt specific source (AltManager.js) should be semicolon-less.